libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -258,6 +258,9 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 // constructor
 AP_GPS::AP_GPS()
 {
+    static_assert((sizeof(_initialisation_blob) * 9) < (4800 * GPS_BAUD_TIME_MS * 1e-3),
```

Comment:
I've banged my head against the wall and I can't figure it out, what is the `* 9` for?